### PR TITLE
Implement sys.sequences as an empty view to fix issue in DDL Export w…

### PR DESCRIPTION
**Pull request by tanyagupta17**
### Description

Implement sys.sequences as an empty view to fix issue in DDL Export with SSMS

Tested with SSMS 19.3 version and 18.12 version, the DDL export was successful.

Signed-off-by: Tanya Gupta [tanyagp@amazon.com](mailto:tanyagp@amazon.com)

### Issues Resolved

BABEL-5099

### Test Scenarios Covered ###
* **Use case based -** YES

* **Boundary conditions -** NA

* **Arbitrary inputs -** NA

* **Negative test cases -** NA

* **Minor version upgrade tests -** NA

* **Major version upgrade tests -** NA

* **Performance tests -** NA

* **Tooling impact -** NA

* **Client tests -** NA

### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: test/JDBC/upgrade/latest/schedule [hunk 1]
```diff
@@ -544,3 +544,5 @@ BABEL-3401
 BABEL-3820
```
Comment (shah-nirmit) [inline test/JDBC/upgrade/latest/schedule]: Blocking: Please Check if we need to add DDL export tests.

Comment (tanyagupta17) [reply test/JDBC/upgrade/latest/schedule]: As per our offline discussion: 
This is an empty view, adding a test case would return only empty view and we won't be able to generate the sequences.Neither are we adding the object in. SMO_script as it won't create any difference and is redundant for now.
Once the view is actually implemented, the DDL export tests should be added appropriately.

**Review by rohit01010 — APPROVED** [review]

File: contrib/babelfishpg_tsql/sql/sys_views.sql [hunk 2]
```diff
@@ -3378,4 +3378,25 @@ AS SELECT
...
+AS SELECT 
+    so.*,
+    CAST(0 as sys.sql_variant) AS start_value
+    , CAST(0 as sys.sql_variant) AS increment
+    , CAST(0 as sys.sql_variant) AS minimum_value
+    , CAST(0 as sys.sql_variant) AS maximum_value
+    , CAST(0 as sys.BIT) AS is_cycling
+    , CAST(0 as sys.BIT) AS is_cached
+    , CAST(0 as INT) AS cache_size
+    , CAST(0 as sys.TINYINT) AS system_type_id
```
Comment (thephantomthief) [inline contrib/babelfishpg_tsql/sql/sys_views.sql]: Although T-SQL specs say this should be tinyint, we should still keep this as int. The reason being that in future when we implement this view, we would most certainly identify this as an OID and we map OIDs to int. For example, sys.types has a column system_type_id which indicates the type oid and is of type int.

File: test/JDBC/input/views/sys_sequences-vu-prepare.sql [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    SELECT * FROM sys.sequences
+GO
+
+CREATE FUNCTION sys_sequences_test_func()
+RETURNS INT
+AS
+BEGIN
+    RETURN (SELECT COUNT(*) FROM sys.sequences)
+END
+GO
```
Comment (thephantomthief) [inline test/JDBC/input/views/sys_sequences-vu-prepare.sql]: Nit: Better to have new line at the end

File: test/JDBC/input/views/sys_sequences-vu-verify.sql [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+GO
+
+SELECT
+SCHEMA_NAME(seq.schema_id) AS [Schema],
+seq.name AS [Name]
+FROM
+sys.sequences AS seq
+ORDER BY
+[Schema] ASC,[Name] ASC
+GO
```
Comment (thephantomthief) [inline test/JDBC/input/views/sys_sequences-vu-verify.sql]: Nit: Better to have new line at the end

File: contrib/babelfishpg_tsql/sql/sys_views.sql (hunk 2 above)
Comment (tanyagupta17) [reply contrib/babelfishpg_tsql/sql/sys_views.sql]: Changed from sys.TINYINT to INT as our expected OID could be larger than 255

File: test/JDBC/input/views/sys_sequences-vu-prepare.sql (hunk 3 above)
Comment (tanyagupta17) [reply test/JDBC/input/views/sys_sequences-vu-prepare.sql]: Done

File: test/JDBC/input/views/sys_sequences-vu-verify.sql (hunk 4 above)
Comment (tanyagupta17) [reply test/JDBC/input/views/sys_sequences-vu-verify.sql]: Done

**Review by rohit01010 — APPROVED** [review]

**Review by thephantomthief — APPROVED** [review]

**Review by shah-nirmit — APPROVED** [review]

**Review by shardgupta — APPROVED** [review]

**Pull request merged**